halloffame/halloffame.php

```diff
@@ -0,0 +1,49 @@
+<?php
```

Comment:
I'm very confused about this file! In fact, it seems like you have another `halloffame` directory inside your main one? And some of the files are duplicated? This needs explaining or re-organising into one main file with no duplicated files.